services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return [
+        {"row": ticket.row, "seat": ticket.seat}
+        for ticket in get_movie_session_by_id(movie_session_id).tickets.all()
+    ]
```

Comment:
Retrieve required values without list comprehension but rather using Django ORM and `values` or `values_list` for better efficiency